src/site/antora/modules/ROOT/partials/manual/api-intro.adoc

```diff
@@ -0,0 +1,72 @@
+////
+    Licensed to the Apache Software Foundation (ASF) under one or more
+    contributor license agreements.  See the NOTICE file distributed with
+    this work for additional information regarding copyright ownership.
+    The ASF licenses this file to You under the Apache License, Version 2.0
+    (the "License"); you may not use this file except in compliance with
+    the License.  You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+////
+
+To write logs, you need a `Logger` instance which you will retrieve from the `LogManager`.
+You can use the `Logger` instance to write logs by using methods like `info()`, `warn()`, `error()`, etc.
+These methods are named after the _log levels_ they represent, a way to categorize log events by severity.
+The log message can also contain placeholders written as `{}` that will be replaced by the arguments passed to the method.
```

Comment:
Maybe:

> The **log message** can also be specified by providing a format string containing `{}` placeholders and an array of arguments.

Users use **log message** and **log event** interchangeably. It would be nice to explain at the beginning that:

* a "log message" is something the user provides explicitly,
* a "log event" is the message enriched with all sorts of meta-data.